src/zoid/card-fields/component.jsx

```diff
@@ -158,7 +173,7 @@ export const getCardFieldsComponent: () => CardFieldsComponent = memoize(
         },
 
         autoResize: {
-          height: true,
+          height: false,
```

Comment:
@sebastianfdz and I were not change this value conditionally since `autoResize` does not accept a callback. We decided to default this to `false` instead of `true` for all renders. We tested the default integration and it is working as expected and there are no breaking changes.